.cspell.json

```diff
@@ -963,6 +986,7 @@
     "OMPL",
     "oneshot",
     "ONNX",
+    "openadkit",
```

Comment:
I feel it's better to rename the Docker image name to `open-ad-kit`, but it's okay for now.